Remove instances of "should". (#32980)
grafana/grafana: contribute/style-guides/documentation-style-guide.md

### File naming conventions

> - Files that are displayed in the help system have names that are all lowercase, no spaces. Use hyphens instead of spaces. Example: glossary.md

Reviewer: ```suggestion
- Files in the help system should have names that are all lowercase, no spaces. Use hyphens instead of spaces. Example: glossary.md
```

> - Documentation file names match the title. **Note:** This only applies to new files at this time. Do not change the names of older files unless directed to do so.

Reviewer: ```suggestion
- Documentation file names should match the title. **Note:** This only applies to new files at this time. Do not change the names of older files unless directed to do so.
```

> - Internal reference file names are be all uppercase except the file extension. Example: CONTRIBUTING.md

Author: ```suggestion
- Internal reference file names are all uppercase except the file extension. Example: CONTRIBUTING.md
```

> - Image file names are descriptive and unique. Also, add the software version number that the image applies to or the screenshot was taken in. Example: share-dashboard-link-7-3.png

Reviewer: Changes make sense.

> - Image file names are be descriptive and unique. Also, add the software version number that the image applies to or the screenshot was taken in. Example: share-dashboard-link-7-3.png

Author: ```suggestion
- Image file names are descriptive and unique. Also, add the software version number that the image applies to or the screenshot was taken in. Example: share-dashboard-link-7-3.png
```

- Task topic headings start with a verb.
  - Write a query
  - Create a dashboard

> - Concept and reference topic headings are nouns or gerunds. Examples: Contributing to docs, Visualizations, Style guide

Reviewer: The sentence should stay as is.

Author: Please use suggestions so it is easier to apply the change.

> - Concept and reference topic headings are nouns or gerunds. Examples: Contributing to docs, Visualizations, Style guide

Author: ```suggestion
- Concept and reference topic headings should be nouns or gerunds. Examples: Contributing to docs, Visualizations, Style guide
```

> - Concept and reference topic headings are be nouns or gerunds. Examples: Contributing to docs, Visualizations, Style guide

Reviewer 2: ```suggestion
- Concept and reference topic headings are nouns or gerunds. Examples: Contributing to docs, Visualizations, Style guide
```

#### Heading don'ts

- Avoid stacked headings, which is following one heading with another heading.

> - Avoid skipping heading levels. For example, an h1 is followed by an h2 rather than an h3.

Reviewer: The change makes sense.

- Avoid having just one lower-level heading. For example, h1, h2, h2, h3, h3, h2, h2 is a good order. Do not go h1, h2, h3, h2, h3, h2.
- Avoid using hyphens in headings.
- Do not include parenthetical words like (Important!) in headings.

#### Step-by-step headings

> In most cases, headings are not numbered steps.

Reviewer: The change makes sense.

> In most cases, headings are not be numbered steps.

Reviewer 2: ```suggestion
In most cases, headings aren't numbered steps.
```

### Images

- Preferred format is .png

> - File extension is all lowercase.

Reviewer: The sentence should stay as is.

> - File extension is all lowercase.

Author: ```suggestion
- File extension should be all lowercase.
```

- Preferred DPI is 72.
- Assume all graphics will be exclusively viewed on the web.
- Maximum image size is 3840px X 2160px.

> - Screenshots are readable, but not too large.

Reviewer: The sentence should stay as is.

> - Screenshots are readable, but not too large.

Author: ```suggestion
- Make screenshots readable, but not too large.
```

Here are a few general rules about unordered lists. For more guidance, refer to [Lists](https://developers.google.com/style/lists) in the [Google developer style guide](https://developers.google.com/style/).

- Begin list items with a capital letter unless there is a strong reason not to. For example, you are listing case-sensitive parameters.

> - End list items with periods if they are complete sentences. If one item in a list ends with a period, then apply periods to all of them.

Reviewer: Changes make sense.

#### Notes

> Notes provide additional information for the user to be aware of. For example:

Reviewer: ```suggestion
Notes provide additional information the user should be aware of. For example:
```

#### Cautions

> Cautions warn the user to proceed with caution. Use cautions to emphasize the potential downside of a course of action.

Reviewer: This is an example, where the re-wording makes sense.
